code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -21,6 +21,11 @@
 /mob/living/simple_animal/hostile/blob/blob_act()
 	return
 
+/mob/living/simple_animal/hostile/blob/CanPass(atom/movable/mover, turf/target, height=0)
```

Comment:
more accurately, I care, but my care is all hate and this isn't the PR to remove the height arg from CanPass entirely